src/ansys/fluent/core/__main__.py

```diff
@@ -0,0 +1,33 @@
+import os
+import sys
+
+from ansys.fluent.core.launcher.launcher import launch_fluent
+from ansys.fluent.core.session_solver import Solver
+
+
+def setup_for_fluent(version: str, mode: str):
```

Comment:
@hpohekar Let's change the signature to `setup_for_fluent(*args, **kwargs)`, so we can update the Fluent code without affecting PyFluent.

Reply:
Yes, sure.

Reply:
Done.